sphinx/ext/todo.py

```diff
@@ -209,6 +210,17 @@ def depart_todo_node(self, node):
     self.depart_admonition(node)
 
 
+def latex_visit_todo_node(self, node):
+    # type: (nodes.NodeVisitor, todo_node) -> None
+    title = node.pop(0).astext().translate(tex_escape_map)
+    self.body.append('\n\\begin{sphinxadmonition}{note}{%s:}' % title)
```

Comment:
Good point. Indeed, we usually use `u` suffix to represent strings. I will update this.

Note: In py2, the strings contain only ASCII chars are treated as both `str` and `unicode` objects. So it is not needed to use `u` suffix to represent them (we have to use it if the string contains non-ASCII chars). But I think using the suffix always is simple and confuse-less.